### DISCUSSION TITLE
Send email to leads instead of users

### DISCUSSION THREAD

**Discussion by realJogicodes** [pocketbase/pocketbase#4343] [Q&A]
Hello

for WIP projects, I like to have a landing page where users can enter their email, nothing else and then get notified when the app is ready to accept users. For market demand validation purposes.

In pocketbase, can I send email to collections that are not users?

Because say I make a collection 'leads' that only stores an email, I'd like to do the same thing as I would do with a new user that did signup with password and send them a verification token and set the lead to verfied=true after click

My thought is currently to do that using the user collection and assigning a random password on the server and then invite people to pick a password uing special reset logic, but that's "hacky", maybe there's a better way where a collection that's not users can send emails and respond to link clicks server side?

**Comment by ganigeorgiev** [comment]
> In pocketbase, can I send email to collections that are not users?

Yes, you should be able to send programmatically emails to any valid email address(es):
- Go: https://pocketbase.io/docs/go-sending-emails/
- JS: https://pocketbase.io/docs/js-sending-emails/

There is no built in mechanism that will convert your "leads" base collection into "users" auth collection. You can do that programmatically with [migration scripts](https://pocketbase.io/docs/go-migrations/) or custom routes but in my opinion registering the leads as regular unverified users with a random password seems easier.
